tools/cppcheck-conda-recipe/meta.yaml

```diff
@@ -3,10 +3,12 @@ package:
   version: '1.80'
 source:
   fn: 1.80.tar.gz
+  sha256: 20863db018d69c33648bdedcdc9d81d818b9064cc4333f0d4dc45e114bd0f000
   url: https://github.com/danmar/cppcheck/archive/1.80.tar.gz
 
 build:
   number: 3
+  string: 1
```

Comment:
I guess the purpose is to keep the `string` and `number` identical, 4 in this case. The cppcheck recipe has undergone already some changes, including yours, it becomes 4.